### DISCUSSION TITLE
Is the project going to go live in end August?

### DISCUSSION THREAD

**Discussion by praveenkrishna0512** [google/flutter-mediapipe#6] [Q&A]
I saw that the original timeline is structured such that the project will release in end-August. Thus, I am wondering if this is still the case, as we are looking for mediapipe integration for flutter asap.

**Comment by PaulTR** [comment]
We were younger then, more optimistic :P It's being worked on, but we don't have a set release date.

**Comment by PaulTR** [comment]
Join this group: I'll send out an email when it's ready: https://groups.google.com/g/mediapipe-solutions-external-announce

**Comment by UsamaHameed1** [comment]
Will the project flutter media pipe package be available in the coming days?
